src/amcrest/log.py

```diff
@@ -27,3 +27,42 @@ def log_show(self, start_time, end_time):
             '{0}&condition.EndTime={1}'.format(start_time, end_time)
         )
         return ret.content.decode('utf-8')
+
+    def log_find_start(self, start_time, end_time):
+        ret = self.command(
+            'log.cgi?action=startFind'
+            '&condition.StartTime={0}&condition.EndTime={1}'
+            .format(
+                start_time.strftime('%Y-%m-%d %H:%M:%S'),
+                end_time.strftime('%Y-%m-%d %H:%M:%S')))
+
+        return ret.content.decode('utf-8')
+
+    def log_find_next(self, token):
+        ret = self.command('log.cgi?action=doFind&token={0}&count=100'
```

Comment:
@maximvelichko the parameter `count=100` could be passed as a parameter into this method? So this way the user could modify it or is it a hardcoded value that you found to work better?